src/components/Map/types.ts

```diff
@@ -0,0 +1,22 @@
+import type {MapComponentSchema} from '@open-formulieren/types';
+
+export type Coordinates = [number, number];
+
+type PointGeometry = {
+  type: 'Point';
+  coordinates: Coordinates;
+};
```

Comment:
```suggestion
interface PointGeometry {
  type: 'Point';
  coordinates: Coordinates;
};
```
or is there a particular reason why you use a `type` instead of `interface`?